test/sync.js

```diff
@@ -896,19 +907,21 @@ tape('sync: destroy during sync is reflected in peer state', function (t) {
   })
 })
 
-tape('sync: 200 photos', function (t) {
+tape.skip('sync: 200 photos', function (t) {
```

Comment:
do we intend to skip this test?

Reply:
Good catch! I have a bunch of temporary commits on HEAD right now to debug these travis failures. I'll revert them before I push.